src/service/standard-actions-impl.js

```diff
@@ -87,15 +100,57 @@ export class StandardActions {
    * @param {!./action-impl.ActionInvocation} invocation
    */
   handleHide(invocation) {
+    // TODO(alanorozco, #7753) use 'hidden' attribute for AMP elements
     const target = dev().assertElement(invocation.target);
+
     this.resources_.mutateElement(target, () => {
       if (target.classList.contains('i-amphtml-element')) {
         target./*OK*/collapse();
       } else {
         toggle(target, false);
+        target.setAttribute('hidden', '');
       }
     });
   }
+
+  /**
+   * Handles "show" action. This is a very simple action where "display: none"
+   * is removed from the target element.
+   * @param {!./action-impl.ActionInvocation} invocation
+   */
+  handleShow(invocation) {
+    // TODO(alanorozco, #7753) use 'hidden' attribute for AMP elements
+    const target = dev().assertElement(invocation.target);
+
+    user().assert(isHidden(target),
```

Comment:
Actually there's no need to check for computedStyle since the inline style will take priority in its result.